Testing/SpecFlow/SpecFlowRAP/SpecFlowCalculator.Specs/Hooks.cs

```diff
@@ -0,0 +1,28 @@
+﻿
+namespace SpecFlowRAP.StepDefinitions
+{
+
+    [Binding]
+    public static class Hooks
+    {
+        // For additional details on SpecFlow hooks see http://go.specflow.org/doc-hooks
+
+        [BeforeFeature]
+        public static void BeforeTestRun(FeatureContext featureContext)
+        {
+            DIContainer.ConfigureServices();
+
+            RAPStepDefinitions rAPStepDefinitions = new RAPStepDefinitions(featureContext);
```

Comment:
```suggestion
            RAPStepDefinitions rapStepDefinitions = new RAPStepDefinitions(featureContext);
```

Mogelijk camelcase gebruiken inplaats van pascal omdat deze toch private local is.